src/domain/event/partial/OccurrenceInfo.tsx

```diff
@@ -7,63 +7,75 @@ import clockIcon from '../../../assets/icons/svg/clock.svg';
 import calendarIcon from '../../../assets/icons/svg/calendar.svg';
 import locationIcon from '../../../assets/icons/svg/location.svg';
 import personIcon from '../../../assets/icons/svg/person.svg';
-import Icon from '../../../common/components/icon/Icon';
 import { formatOccurrenceTime } from '../EventUtils';
 import { childByIdQuery_child_enrolments_edges_node_occurrence as OccurrenceType } from '../../api/generatedTypes/childByIdQuery';
 import { occurrenceQuery_occurrence as OccurrenceQueryType } from '../../api/generatedTypes/occurrenceQuery';
 import { formatTime, newMoment } from '../../../common/time/utils';
 import { DEFAULT_DATE_FORMAT } from '../../../common/time/TimeConstants';
+import InfoItem from './InfoItem';
 
-interface OccurrenceInfoProps {
+type Props = {
   className?: string;
   occurrence: OccurrenceType | OccurrenceQueryType;
-}
+  show?: string[];
+};
+
+type InfoItem = {
+  id: string;
+  icon: string;
+  iconAlt?: string;
+  label: string;
+};
 
-const OccurrenceInfo: FunctionComponent<OccurrenceInfoProps> = ({
+const OccurrenceInfo: FunctionComponent<Props> = ({
   className,
   occurrence,
+  show = ['time', 'duration', 'participants', 'venue'],
 }) => {
   const { t } = useTranslation();
+
+  const infoItems: InfoItem[] = [
+    {
+      id: 'time',
+      icon: calendarIcon,
+      label: formatTime(newMoment(occurrence.time), DEFAULT_DATE_FORMAT),
+    },
+    {
+      id: 'duration',
+      icon: clockIcon,
+      label: formatOccurrenceTime(occurrence.time, occurrence.event.duration),
+    },
+    {
+      id: 'participants',
+      icon: personIcon,
+      label: t(
+        `event.participantsPerInviteEnum.${occurrence.event.participantsPerInvite}`
+      ),
+    },
+    {
+      id: 'venue',
+      icon: locationIcon,
+      label: occurrence.venue.name || '',
+    },
+  ];
+
+  console.log('classname');
```

Comment:
This could use template string.

Reply:
This one should not have been part of the pr 😅